files/package.json

```diff
@@ -12,11 +12,12 @@
     "test": "ember test"
   },
   "devDependencies": {
-    "@glimmer/application": "^0.7.2",
-    "@glimmer/application-pipeline": "^0.8.0",
-    "@glimmer/blueprint": "^0.5.0",
+    "@glimmer/application": "^0.8.0-alpha.9",
+    "@glimmer/application-pipeline": "^0.9.0",
+    "@glimmer/blueprint": "^0.6.0",
```

Comment:
I've preemptively updated the `@glimmer/blueprin` version to `^0.6.0` in anticipation of the bump which follows this PR being merged